### DISCUSSION TITLE
Updates

### DISCUSSION THREAD

**Discussion by Firecul** [ThioJoe/YT-Spam-Lists#156] [General]
I take it it'd be better for me just to push updates daily/as needed rather than waiting to do it once a week @ThioJoe ?
I was thinking daneel6672 was going to continue posting very often so batching them would be the better thing but if he has slowed down daily would be better I guess.

**Comment by ThioJoe** [comment]
Yea you can just push it as often as you want. Sometimes I even push multiple per day

**Reply by Firecul** [reply]
Ok, I'll just do that then.
